Makefile

```diff
@@ -83,7 +80,9 @@ $(TARBALL): import $(BUILD_DIR)
 
 .PHONY: build-app
 build-app: $(BUILD_DIR) images
-	$(TELE) build -o build/installer.tar $(TELE_BUILD_OPTIONS) $(EXTRA_GRAVITY_OPTIONS) resources/app.yaml
+	sed -i "s/version: \"0.0.0+latest\"/version: \"$(RUNTIME_VERSION)\"/" resources/app.yaml
+	$(TELE) build -f -o build/installer.tar $(TELE_BUILD_OPTIONS) $(EXTRA_GRAVITY_OPTIONS) resources/app.yaml
+	if grep $(RUNTIME_VERSION) resources/app.yaml; then sed -i "s/version: \"$(RUNTIME_VERSION)\"/version: \"0.0.0+latest\"/" resources/app.yaml; fi
```

Comment:
you don't really need to `grep` here... right? :thinking: 
`sed` will not execute anything if it doesn't match the first part of the RegEx anyway.